assets/src/js/Components/ArchivePicker/ArchivePicker.js

```diff
@@ -324,13 +325,13 @@ export default function ArchivePicker({view = ADMIN_VIEW}) {
       dispatch({type: ACTIONS.ADDED_IMAGE_TO_POST});
       await timeout(1000);
       dispatch({type: ACTIONS.CLOSE_SIDEBAR});
-      document.querySelector('.media-modal-close').click();
+      document.querySelector('.media-modal-icon').click();
```

Comment:
I don't think that this selector is specific enough 🤔 

<img width="338" alt="Screenshot 2024-10-07 at 15 08 11" src="https://github.com/user-attachments/assets/9727f00c-9b16-41fb-9ee1-09c594b79a66">